### PR TITLE
Add data plane test fib/ip/ipfwd tests to onboarding PR checker

### PR DESCRIPTION
**Pull request by xwjiang-ms**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

#### What is the motivation for this PR?
Elastictest performs well in distribute running PR test in multiple KVMs, which support us to add more test scripts to PR checker.
But some traffic test using ptfadapter can't be tested on KVM platform, we need to skip traffic test if needed
#### How did you do it?
1. Add fib/ip/ipfwd test to onboarding test for pilot and skip traffic test
2. Add test to skip pr test yaml
3. Sort test scripts in pr_test/skip_pr_test yaml

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions_skip_traffic_test.yaml [hunk 1]
```diff
@@ -116,3 +116,50 @@ everflow/test_everflow_testbed.py:
...
+    conditions:
+      - "asic_type in ['vs']"
+
+# Temporary add because conditional mark would only match longest prefix
+ip/test_ip_packet.py::TestIPPacket::test_forward_ip_packet_with_0xffff_chksum_tolerant:
+  skip_traffic_test:
+    reason: "Skip traffic test for KVM testbed"
+    conditions:
+      - "asic_type in ['vs']"
+
```
Comment (opcoder0) [inline tests/common/plugins/conditional_mark/tests_mark_conditions_skip_traffic_test.yaml:157]: I see in `line 138` the entire test module has been set for skip. Is there a reason to choose these two tests specifically ? Won't they be skipped anyway as part of the module skip ?

**Review by opcoder0 — COMMENTED** [review]
LGTM. Just a couple of questions

I see some ixia tests have been marked with topology marker `physical`. Do those get skipped on `vs` automatically? I didn't see them in the `pr_test_skip_scripts.yml`

File: tests/common/plugins/conditional_mark/tests_mark_conditions_skip_traffic_test.yaml (hunk 1 above)
Comment (xwjiang-ms) [reply tests/common/plugins/conditional_mark/tests_mark_conditions_skip_traffic_test.yaml:157]: Currently conditional mark would only match longest prefix, `ip/test_ip_packet.py::TestIPPacket::test_forward_ip_packet_with_0xffff_chksum_drop` and `ip/test_ip_packet.py::TestIPPacket::test_forward_ip_packet_with_0xffff_chksum_tolerant` were defined in `tests\common\plugins\conditional_mark\tests_mark_conditions.yaml`, so the module skip won't be matched in these 2 testcases. Yutong will fix it, but we need a temporary workaround

**Review by opcoder0 — APPROVED** [review]

**Comment by xwjiang-ms** [comment]
> LGTM. Just a couple of questions
> 
> I see some ixia tests have been marked with topology marker `physical`. Do those get skipped on `vs` automatically? I didn't see them in the `pr_test_skip_scripts.yml`

That's another logic of static, take t0 as example, if a test script has the topology mark neither t0, any or dualtor, then it won't be included in t0 calculation, so since ixia test has the mark of `tgen` topo, we needn't add to skip yml

**Comment by xwjiang-ms** [comment]
mark as draft since will use another temporary fixture to skip traffic test

**Review by wangxin — APPROVED** [review]

**Comment by sdszhang** [comment]
@xwjiang-ms @bingwang-ms Will this PR be integrated into 202405?

**Comment by xwjiang-ms** [comment]
> @xwjiang-ms @bingwang-ms Will this PR be integrated into 202405?

No, it won't, only master branch would have `skip_traffic_test` fixture

**Pull request merged**